schemas/context/language.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/language",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Language",
+  "type": "object",
+  "description":
+    "The system of communication used by a particular person, country or community.",
+  "definitions": {
+    "language": {
+      "properties": {
+        "xdm:format": {
+          "title": "Format Identifier",
+          "type": "string",
+          "description":
+            "Standardized nomenclature used to classify the language",
+          "meta:enum": {
```

Comment:
I'm not a fan of this at all. We should use BCP-47 across the board, except in cases where we have to maintain compatibility with XMP, which predates BCP-47.

Reply:
@trieloff , @lrosenthol : We follow ISO-639-1 codes in Campaign for language specification and hence need this functionality. Please comment on how can we incorporate this in the best way.

Reply:
Within XDM: use BCP-47
In the Adobe Campaign extensions of XDM: use whatever you want (although my recommendation would be BCP-47, as it makes things easier)
In the Adobe Campaign implementation: use whatever you want and map it to BCP-47.

Reply:
While I was pushing for supporting ISO-636-1 earlier as well, I agree with this point now.
@aarjain We should use a mapper to send data in the BCP-47 format. We finally want to send data in a "standard" format.

Reply:
Made modifications to use BCP 47 as the standard for specifying language codes.